astroid/__pkginfo__.py

```diff
@@ -23,7 +23,7 @@
 extras_require = {}
 install_requires = [
     "lazy_object_proxy==1.4.*",
-    "six==1.12",
+    "six==1.13",
```

Comment:
Should be `six~=1.13` otherwise, it will break the whole world again.
I've released Conan `1.7.6`, `1.8.5`, `1.9.3`, `1.10.3`, `1.11.3`, `1.12.4`, `1.13.4`, `1.14.6`, `1.15.5`, `1.16.2`, `1.17.3`, `1.18.6` and `1.19.4` fixing this, so please, not again.

Reply:
done